{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intel MKL WARNING: Support of Intel(R) Streaming SIMD Extensions 4.2 (Intel(R) SSE4.2) enabled only processors has been deprecated. Intel oneAPI Math Kernel Library 2025.0 will require Intel(R) Advanced Vector Extensions (Intel(R) AVX) instructions.\n",
      "Intel MKL WARNING: Support of Intel(R) Streaming SIMD Extensions 4.2 (Intel(R) SSE4.2) enabled only processors has been deprecated. Intel oneAPI Math Kernel Library 2025.0 will require Intel(R) Advanced Vector Extensions (Intel(R) AVX) instructions.\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "os.chdir('')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "fulltext_randomsample = pd.read_csv('fulltext_randomsample.csv', index_col=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>corpusid</th>\n",
       "      <th>content</th>\n",
       "      <th>arxiv</th>\n",
       "      <th>mag</th>\n",
       "      <th>acl</th>\n",
       "      <th>pubmed</th>\n",
       "      <th>pubmedcentral</th>\n",
       "      <th>dblp</th>\n",
       "      <th>doi</th>\n",
       "      <th>rn</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>24941224</td>\n",
       "      <td>{\"source\": {\"pdfurls\": null, \"pdfsha\": \"48fcb8...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2.125976e+09</td>\n",
       "      <td>NaN</td>\n",
       "      <td>16242152.0</td>\n",
       "      <td>7094468.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1016/j.jmb.2005.09.012</td>\n",
       "      <td>11907</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>234022863</td>\n",
       "      <td>{\"source\": {\"pdfurls\": [\"https://web.archive.o...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3.126698e+09</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1155/2021/8849303</td>\n",
       "      <td>16259</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>220409365</td>\n",
       "      <td>{\"source\": {\"pdfurls\": null, \"pdfsha\": \"b2d971...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3.041079e+09</td>\n",
       "      <td>NaN</td>\n",
       "      <td>32733529.0</td>\n",
       "      <td>7363983.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.3389/fpls.2020.01057</td>\n",
       "      <td>19987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>56102549</td>\n",
       "      <td>{\"source\": {\"pdfurls\": null, \"pdfsha\": \"805fb5...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2.808589e+09</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.5194/amt-11-6271-2018</td>\n",
       "      <td>23430</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>231665585</td>\n",
       "      <td>{\"source\": {\"pdfurls\": null, \"pdfsha\": \"306be7...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>33474627.0</td>\n",
       "      <td>8238723.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1007/s00420-020-01619-1</td>\n",
       "      <td>33353</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>53210972</td>\n",
       "      <td>{\"source\": {\"pdfurls\": null, \"pdfsha\": \"894e0b...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2.898356e+09</td>\n",
       "      <td>NaN</td>\n",
       "      <td>30397645.0</td>\n",
       "      <td>6203223.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1126/sciadv.aau0244</td>\n",
       "      <td>801983</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>226248203</td>\n",
       "      <td>{\"source\": {\"pdfurls\": null, \"pdfsha\": \"81e354...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3.097526e+09</td>\n",
       "      <td>NaN</td>\n",
       "      <td>33144666.0</td>\n",
       "      <td>7642341.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1038/s42003-020-01364-8</td>\n",
       "      <td>804374</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>263285864</td>\n",
       "      <td>{\"source\": {\"pdfurls\": null, \"pdfsha\": \"cd9426...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>37841205.0</td>\n",
       "      <td>10570437.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.3389/fchem.2023.1272034</td>\n",
       "      <td>805552</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>237503175</td>\n",
       "      <td>{\"source\": {\"pdfurls\": null, \"pdfsha\": \"ab0a8f...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>34527066.0</td>\n",
       "      <td>8437591.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1155/2021/7924400</td>\n",
       "      <td>807654</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>119489650</td>\n",
       "      <td>{\"source\": {\"pdfurls\": null, \"pdfsha\": \"02f3a8...</td>\n",
       "      <td>1807.02866</td>\n",
       "      <td>3.104857e+09</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.3847/2041-8213/aad813</td>\n",
       "      <td>845432</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>100 rows × 10 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     corpusid                                            content       arxiv  \\\n",
       "0    24941224  {\"source\": {\"pdfurls\": null, \"pdfsha\": \"48fcb8...         NaN   \n",
       "1   234022863  {\"source\": {\"pdfurls\": [\"https://web.archive.o...         NaN   \n",
       "2   220409365  {\"source\": {\"pdfurls\": null, \"pdfsha\": \"b2d971...         NaN   \n",
       "3    56102549  {\"source\": {\"pdfurls\": null, \"pdfsha\": \"805fb5...         NaN   \n",
       "4   231665585  {\"source\": {\"pdfurls\": null, \"pdfsha\": \"306be7...         NaN   \n",
       "..        ...                                                ...         ...   \n",
       "95   53210972  {\"source\": {\"pdfurls\": null, \"pdfsha\": \"894e0b...         NaN   \n",
       "96  226248203  {\"source\": {\"pdfurls\": null, \"pdfsha\": \"81e354...         NaN   \n",
       "97  263285864  {\"source\": {\"pdfurls\": null, \"pdfsha\": \"cd9426...         NaN   \n",
       "98  237503175  {\"source\": {\"pdfurls\": null, \"pdfsha\": \"ab0a8f...         NaN   \n",
       "99  119489650  {\"source\": {\"pdfurls\": null, \"pdfsha\": \"02f3a8...  1807.02866   \n",
       "\n",
       "             mag  acl      pubmed  pubmedcentral dblp  \\\n",
       "0   2.125976e+09  NaN  16242152.0      7094468.0  NaN   \n",
       "1   3.126698e+09  NaN         NaN            NaN  NaN   \n",
       "2   3.041079e+09  NaN  32733529.0      7363983.0  NaN   \n",
       "3   2.808589e+09  NaN         NaN            NaN  NaN   \n",
       "4            NaN  NaN  33474627.0      8238723.0  NaN   \n",
       "..           ...  ...         ...            ...  ...   \n",
       "95  2.898356e+09  NaN  30397645.0      6203223.0  NaN   \n",
       "96  3.097526e+09  NaN  33144666.0      7642341.0  NaN   \n",
       "97           NaN  NaN  37841205.0     10570437.0  NaN   \n",
       "98           NaN  NaN  34527066.0      8437591.0  NaN   \n",
       "99  3.104857e+09  NaN         NaN            NaN  NaN   \n",
       "\n",
       "                           doi      rn  \n",
       "0    10.1016/j.jmb.2005.09.012   11907  \n",
       "1         10.1155/2021/8849303   16259  \n",
       "2      10.3389/fpls.2020.01057   19987  \n",
       "3     10.5194/amt-11-6271-2018   23430  \n",
       "4   10.1007/s00420-020-01619-1   33353  \n",
       "..                         ...     ...  \n",
       "95      10.1126/sciadv.aau0244  801983  \n",
       "96  10.1038/s42003-020-01364-8  804374  \n",
       "97  10.3389/fchem.2023.1272034  805552  \n",
       "98        10.1155/2021/7924400  807654  \n",
       "99    10.3847/2041-8213/aad813  845432  \n",
       "\n",
       "[100 rows x 10 columns]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "fulltext_randomsample"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'\\npH-dependent Conformational Flexibility of the SARS-CoV Main Proteinase (M pro ) Dimer: Molecular Dynamics Simulations and Multiple X-ray Structure Analyses\\n\\n\\nJinzhi Tan \\nCenter for Drug Discovery and Design\\nGraduate School of Chinese Academy of Sciences Shanghai Institutes for Biologi-cal Sciences\\nState Key Laboratory of Drug Research\\nShanghai Institute of Materia Medica\\nChinese Academy of Sciences\\n555 Zu Chong Zhi Road, Zhangjiang Hi-Tech Park201203ShanghaiChina\\n\\nInstitute of Biochemistry Center for Structural and Cell Biology in Medicine\\nUniversity of Lü beck\\nRatzeburger Allee 160 23538 Lü beckGermany\\n\\nH G Koen \\nVerschueren \\nInstitute of Biochemistry Center for Structural and Cell Biology in Medicine\\nUniversity of Lü beck\\nRatzeburger Allee 160 23538 Lü beckGermany\\n\\n† \\nKanchan Anand \\nInstitute of Biochemistry Center for Structural and Cell Biology in Medicine\\nUniversity of Lü beck\\nRatzeburger Allee 160 23538 Lü beckGermany\\n\\nJianhua Shen \\nCenter for Drug Discovery and Design\\nGraduate School of Chinese Academy of Sciences Shanghai Institutes for Biologi-cal Sciences\\nState Key Laboratory of Drug Research\\nShanghai Institute of Materia Medica\\nChinese Academy of Sciences\\n555 Zu Chong Zhi Road, Zhangjiang Hi-Tech Park201203ShanghaiChina\\n\\nMaojun Yang \\nInstitute of Biochemistry Center for Structural and Cell Biology in Medicine\\nUniversity of Lü beck\\nRatzeburger Allee 160 23538 Lü beckGermany\\n\\nYechun Xu \\nCenter for Drug Discovery and Design\\nGraduate School of Chinese Academy of Sciences Shanghai Institutes for Biologi-cal Sciences\\nState Key Laboratory of Drug Research\\nShanghai Institute of Materia Medica\\nChinese Academy of Sciences\\n555 Zu Chong Zhi Road, Zhangjiang Hi-Tech Park201203ShanghaiChina\\n\\nZihe Rao \\nTsinghua-IBP Joint Research Group for Structural Biology\\nTsinghua University\\n100084Beijing\\n\\nJanna Bigalke \\nInstitute of Biochemistry Center for Structural and Cell Biology in Medicine\\nUniversity of Lü beck\\nRatzeburger Allee 160 23538 Lü beckGermany\\n\\nBurkhard Heisen \\nInstitute of Biochemistry Center for Structural and Cell Biology in Medicine\\nUniversity of Lü beck\\nRatzeburger Allee 160 23538 Lü beckGermany\\n\\nJeroen R Mesters \\nInstitute of Biochemistry Center for Structural and Cell Biology in Medicine\\nUniversity of Lü beck\\nRatzeburger Allee 160 23538 Lü beckGermany\\n\\nKaixian Chen \\nCenter for Drug Discovery and Design\\nGraduate School of Chinese Academy of Sciences Shanghai Institutes for Biologi-cal Sciences\\nState Key Laboratory of Drug Research\\nShanghai Institute of Materia Medica\\nChinese Academy of Sciences\\n555 Zu Chong Zhi Road, Zhangjiang Hi-Tech Park201203ShanghaiChina\\n\\nXu Shen \\nCenter for Drug Discovery and Design\\nGraduate School of Chinese Academy of Sciences Shanghai Institutes for Biologi-cal Sciences\\nState Key Laboratory of Drug Research\\nShanghai Institute of Materia Medica\\nChinese Academy of Sciences\\n555 Zu Chong Zhi Road, Zhangjiang Hi-Tech Park201203ShanghaiChina\\n\\nHualiang Jiang \\nCenter for Drug Discovery and Design\\nGraduate School of Chinese Academy of Sciences Shanghai Institutes for Biologi-cal Sciences\\nState Key Laboratory of Drug Research\\nShanghai Institute of Materia Medica\\nChinese Academy of Sciences\\n555 Zu Chong Zhi Road, Zhangjiang Hi-Tech Park201203ShanghaiChina\\n\\nRolf Hilgenfeld \\nInstitute of Biochemistry Center for Structural and Cell Biology in Medicine\\nUniversity of Lü beck\\nRatzeburger Allee 160 23538 Lü beckGermany\\n\\n\\nInstitute of Biophysics\\nNational Laboratory of Biomacromolecules\\nChinese Academy of Sciences\\n100101BeijingChina\\n\\npH-dependent Conformational Flexibility of the SARS-CoV Main Proteinase (M pro ) Dimer: Molecular Dynamics Simulations and Multiple X-ray Structure Analyses\\n10.1016/j.jmb.2005.09.012Since January 2020 Elsevier has created a COVID-19 resource centre with free information in English and Mandarin on the novel coronavirus COVID-19. The COVID-19 resource centre is hosted on Elsevier Connect, the company\\'s public news and information website. Elsevier hereby grants permission to make all its COVID-19-related research that is available on the COVID-19 resource centre -including this research content -immediately available in PubMed Central and other publicly funded repositories, such as the WHO COVID database with rights for unrestricted research re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for free by Elsevier for as long as the COVID-19 resource centre remains active. *Corresponding authorSARS-CoV M promolecular dynamics simulationnew crystal formsmultiple X-ray structuresconformational flexibility\\nThe SARS coronavirus main proteinase (M pro ) is a key enzyme in the processing of the viral polyproteins and thus an attractive target for the discovery of drugs directed against SARS. The enzyme has been shown by X-ray crystallography to undergo significant pH-dependent conformational changes. Here, we assess the conformational flexibility of the M pro by analysis of multiple crystal structures (including two new crystal forms) and by molecular dynamics (MD) calculations. The MD simulations take into account the different protonation states of two histidine residues in the substrate-binding site and explain the pH-activity profile of the enzyme. The low enzymatic activity of the M pro monomer and the need for dimerization are also discussed.\\n\\nIntroduction\\n\\nSevere acute respiratory syndrome (SARS) became a global threat due to its rapid transmission and high fatality rate. 1 Up to now, neither an efficacious therapy nor a reliable preventive treatment has become available, although several approaches towards these goals are being tested at the moment.\\n\\nSoon after the initial outbreak, a new coronavirus, SARS-CoV, was discovered as the etiological agent of SARS. [1][2][3] Coronaviruses are positive-strand RNA viruses featuring the largest viral RNA genomes (about 30 kb) found to date. It has been shown that all the protein functions required for SARS coronavirus replication are encoded by the replicase gene. 4,5 This gene encodes two overlapping polyproteins (pp1a and pp1ab), from which the functional proteins are released by extensive proteolytic processing. This is primarily achieved by the 33 kDa main proteinase (M pro ), which is frequently also called 3C-like proteinase (3CL pro ) to indicate a similarity in substrate specificity with the 3C proteinase of picornaviruses. The 3C proteinase of rhinovirus has been selected as a target for the development of drugs against the common cold. 6,7 Similarly, the functional importance of the SARS-CoV M pro in the viral life-cycle makes it a preferred target for discovering anti-SARS drugs. [8][9][10][11] Sequence alignment of the SARS-CoV M pro with orthologues from other coronaviruses indicates that the enzyme is highly conserved, with 40% and 44% sequence identity, respectively, to human CoV (HCoV) 229E M pro and porcine transmissible gastroenteritis virus (TGEV) M pro , the crystal structures of which have been reported. 8,12 Homology modeling for the SARS-CoV M pro has been performed by several groups. 8,9 More recently, the crystal structure of the SARS-CoV M pro has been determined. 13,14 This structure, like those of other CoV M pro s, comprises three domains (Figure 1(a)). Domains I (residues 8-101) and II (residues 102-184) are b-barrels and together resemble the structure of chymotrypsin, whereas domain III (residues 201-306) consists mainly of a-helices. Domains II and III are connected by a long loop (residues 185-200). Located in a cleft between domains I and II, the active site of the M pro comprises a catalytic dyad consisting of the conserved residues Cys145 and His41. A water molecule is hydrogen-bonded to His41 and can be considered the third component of a catalytic triad. In addition, in vitro experiments demonstrated that truncations of domain III reduced significantly or abolished completely the proteolytic activities of the main proteinases of HCoV 229E 15 and TGEV. 12 It was suggested that domain III of M pro is required  40 (c) was prepared by PyMOL. 41 for keeping the proteolytic activity by holding domain II and the long loop (residues 185-200) in a catalytically competent orientation, 12 and/or that its main role is orienting the N-terminal residues that play an important role for the catalytic activity of the enzyme (see below and Yang et al. 13 ).\\n\\nIn all known crystal structures of coronavirus main proteinases, the enzyme exists as a dimer (Figure 1(b)), 8,[12][13][14] and dimerization is also observed in solution at slightly elevated concentrations. 12,[16][17][18] It is believed that the dimer is the enzymatically active species, since the specific activity increases linearly with increasing enzyme concentration. 18 A special feature first discovered for the SARS-CoV M pro (but most probably present in all coronavirus main proteinases) is that in the monoclinic crystals grown at pH 6.0, the two monomers have different conformations around the S1 substrate-binding site, since the loop 138-145, in particular Phe140, as well as Glu166 undergo dramatic conformational rearrangements. As a result, one protomer exists in an active and the other in an inactive conformation. 13 In the latter, the S1 substrate-binding pocket has virtually collapsed as the consequence of the reorientation of Glu166, and the oxyanion hole no longer exists due to the conformational change of residues 138-145 (see Figure 6(a) below). On the other hand, when the crystals are equilibrated at pH 7.6 or pH 8.0, both monomers are in an active conformation. 13 Thus, the coronavirus M pro is a very flexible protein, the conformational state of which appears to depend on the pH value of the medium. This is probably of biological significance, because the viral polyproteins (of which the M pro is a domain before self-release by autocleavage) assemble on the late endosome where local pH tends to be acidic. Here, we analyze the conformational flexibility of the SARS-CoV M pro in detail, and take the pH of the medium into account. We compare the results of molecular dynamics simulations with multiple crystal structures of the free enzyme, including two new crystal forms. In addition to the previously described monoclinic crystal form (space group P2 1 ), we obtained a tetragonal and an orthorhombic crystal form for the SARS-CoV M pro (space groups P4 3 2 1 2 and P2 1 2 1 2). The corresponding refined structures provide valuable information on the flexibility of the enzyme when compared to one another and to the original structure in the monoclinic form. In addition, we describe several independent determinations of the structure in the monoclinic crystal form. The motivation for this is that unless protein crystals diffract to Bragg spacings of 1.5 Å or better, it is normally difficult to identify minor, i.e. less frequently occupied, conformations of flexible parts of a protein from one crystal. However, if different crystals that are grown at slightly deviating conditions are analyzed, one conformation of a flexible surface loop or amino acid side-chain may be selected as the predominant one in one crystal, and another conformation in the other. Thus, independent structural analyses of multiple crystals grown at almost but, in practice, never absolutely identical conditions, will provide useful information on the flexibility of the macromolecule under study. In the work presented here, minor differences in crystallization conditions were introduced by the presence of different weak inhibitors of the SARS-CoV M pro in the crystallization set-ups. However, none of these compounds was found to have bound to the enzyme, as evidenced by the complete absence of even spurious difference density in the substrate-binding site. In our analysis, we focus on the pH-dependent rearrangements observed in the substrate-binding site and investigate the conformational changes that lead to activation/deactivation of the enzyme. We also address the question of why the monomeric proteinase shows little enzymatic activity, whereas the dimer is fully active.\\n\\n\\nResults and Discussion\\n\\n\\nX-ray structures\\n\\nA total of seven X-ray structures of the free SARS-CoV main proteinase have been determined as part of this study. One is derived from newly obtained tetragonal crystals grown at pH 5.9 (space group P4 3 2 1 2, resolution 2.0 Å , R factor 17.5%), the second from orthorhombic crystals obtained at pH 6.6 (space group P2 1 2 1 2, resolution 2.8 Å , R factor 20.8%). The crystallization conditions for the tetragonal form were identical to the previously established ones of the monoclinic form, and in fact, crystals of both forms often appeared simultaneously in the same crystallization droplet. Lower pH and slightly higher concentrations of the precipitant polyethylene glycol (PEG) 6000 (up to 20%) tended to favor the tetragonal form. The orthorhombic crystals were grown using 0.7 M sodium malonate (pH 6.6) as a precipitant. Details of diffraction data collection and structure solution as well as refinement of the new structures are provided in Table 1. The other five structures were determined from crystals of the original monoclinic form that had been grown at pH 6.0. The resolutions of the latter structural models varied between 2.14Å and 2.8 Å , and the crystallographic R factors were between 21.0% and 23.8%. The structures derived from monoclinic crystals confirm the findings reported with the original structure of the enzyme: 13 monomer A is in an active conformation, with the S1 substrate-binding subsite accessible and the oxyanion loop, residues 138-145, in the catalytically competent conformation, whereas both features have collapsed in monomer B of the dimer, which therefore is considered inactive. In contrast to the monoclinic crystals, which feature an M pro dimer in the asymmetric unit, there is only a monomer in the asymmetric unit in the tetragonal (P4 3 2 1 2) and orthorhombic (P2 1 2 1 2) crystal forms, with the dimer being created through crystallographic symmetry. In both of the new crystal forms, which had been obtained at pH 5.9 and 6.6, respectively, the monomers are in the inactive conformation, leading to dimers with both subunits being inactive.\\n\\nApart from the differences in detail in the substrate-binding site, the SARS-CoV M pro dimers as seen in the new structures are very similar to the dimer in the original monoclinic crystals grown at pH 6.0 (Figure 1(c)). From the latter structure, the monomers in the new crystal forms display overall root-mean-square (RMS) deviations for C a atoms (monomers A and B, respectively) of 0.95/0.76 Å (tetragonal form) and 1.10/0.78 Å (orthorhombic form). It is reassuring that the monomers in the new crystal forms, which are in the inactive conformation, are more similar to the inactive monomer B of the dimer in the monoclinic form (second number), than to the active monomer A (first number).\\n\\nThe overall RMS deviation (for C a atoms) of each of the five repetitive monoclinic structures from the original structure of Yang et al. 13 is between 0.3 Å and 0.5 Å (Figure 3(e) and (f)). The crystal structures provide information on the flexibility of the polypeptide chains. The regions exhibiting the largest deviations in both monomers include residues 43-49, 117-122, and 140-144 in domains I and II. Segment 43-49 shows the largest deviations among the A monomers, with RMS values up to 3.5 Å , whereas the most flexible region in monomer B appears to be the segment 140-144, with one structure having a peak of 5.5 Å at residue Phe140. Domain III also shows some flexibility, but this is less focused on a limited number of short segments of the polypeptide chain, with the exception of the C-terminal three or four residues, which have in fact large deviations. The RMS deviations of the tetragonal and orthorhombic structures from the original monoclinic structure (molecule B) follow the same pattern (see Figure 3(f)). These data on polypeptide-chain flexibility are compared below to those obtained from the molecular dynamics (MD) calculations and from atomic temperature factors of the original crystal structure of the SARS-CoV M pro 13 and the new crystal forms.\\n\\n\\nQuality of the MD simulations\\n\\nWe have carried out 10 ns MD simulations of the SARS-CoV M pro dimer at four different protonation states of two histidine residues (His163 and His172) involved in the substrate-binding site ( Table 2). These protonation states were correlated with the pH of the system, and were assumed on the basis of the hydrogen-bonding pattern derived from the crystal structures and chosen to represent all relevant possible combinations (see Materials and Methods for charge assignment). In the cases of the simulations at pH 6.0, 7.6, and 8.0, the starting structure was the (energy-minimized) X-ray structure derived from monoclinic crystals grown at the respective pH. 13 For the state in which His163 is protonated in both monomers of the dimer (here called pH 5.0), no crystal structure was available at the time of the simulation; therefore, the starting model for this was the monoclinic crystal structure at pH 6.0, but with His163 in both protomers positively charged. Since this simulation was carried out, we have obtained the new crystal forms, which presumably have His163 protonated in both monomers. Thus, the new structures provide an experimental check of the predictions derived from the MD calculations. In fact, a comparison of the 10 ns snapshot of the MD simulation at pH 5.0 and the new crystal structures shows that, while there are significant deviations on a residue-by-residue basis in domain III, where some loops have a rather large amplitude of motion during the simulation, the region around the substrate-binding site, including the oxyanion loop 138-145, agrees quite well. The new X-ray Table 2. Protonation states of His163 and His172 in the MD simulation models structures thus confirm the relevance of our MD simulations, which were apparently able to transform one conformation of the SARS-CoV M pro into the other. The nature of these pH-dependent rearrangements will be discussed below.\\nH163(A) H163(B) H172(A) H172(B) pH 5.0 C C C C pH 6.0 K C C C pH 7.6 K K C C pH 8.0 K K K K\\nThe system temperature and the total energy of each simulation model were monitored during the MD simulation and found to converge to stable values. The system temperature was coupled at 300 K and the total energy fluctuated within less than 0.2%. The root-mean-square deviation (RMSD) from the starting structure is an important criterion for the convergence of the system. The RMSDs of the C a atoms are shown in Figure 2, indicating that the whole simulation system appears to have been stable after 2 ns of equilibration.\\n\\n\\nFlexibility of the polypeptide chains\\n\\nTo identify the most flexible parts of the proteinase, we calculated RMS fluctuations (RMSFs) for individual residues, and compared them with the crystallographic results. Figure 3(a) and (b) show the atomic fluctuations averaged over residues for M pro protomers A and B, derived from the 10 ns MD trajectory at pH 6.0. There are two ways of extracting information on residue flexibilities from the crystallographic data. The first is to derive the corresponding RMSF values from the atomic temperature factors (B factors). The experimental B factors are transformed to the RMSF using the formula hDr 2 i iZ 3B i =ð8p 2 Þ, where r 2 i is a coordinate vector of the ith atom. This is shown in Figure 3(c) and (d) for the monoclinic crystal structure at pH 6.0 (molecules A and B). Also, the RMSF values derived from the tetragonal and orthorhombic crystal structures are given in Figure 3(d) (red and green, respectively). It is  \\n\\n\\nSARS-CoV Main Proteinase\\n\\nobvious that the fluctuations are larger in the orthorhombic crystal form, which correlates well with the high Wilson B factor (76 Å 2 ) of these crystals and the relatively low resolution of 2.8 Å .\\n\\nThe second method for obtaining dynamic information from crystal structures is careful analysis of multiple determinations of the same structure, either from crystals in different space groups, or from different crystals in one and the same space group, but grown under slightly different conditions. Figure 3(e) and (f) display the RMS values for both protomer A and B obtained from the multiple X-ray structure determinations carried out with monoclinic crystals grown at pH 6.0. Also, the corresponding RMS values for the tetragonal and orthorhombic crystal structures are included in Figure 3 Figure 1(a)), as well as surface residues 153-155 and 274-277 in each of the two protomers (shaded areas in Figure 3). As will be discussed below, all these loops except the last are associated with the flexibility of the substratebinding pocket. This is also true for the 3 10 -helical segment 45-49, where an insertion is found in SARS-CoV M pro (compared to other coronavirus main proteinases) and which was difficult to model in the X-ray structures due to its flexibility. In addition, most residues involved in the monomermonomer interface of the dimer, such as Ala7, Val125, and Glu290, show reduced fluctuations, while Ser139, Phe140, and Glu166, although contributing to the same dimerization interface, exhibit very high fluctuations, indicating that these residues play a special role (see below). The dimer appears not to be very stable, given the fact that at concentrations below 1 mg/ml, the coronavirus main proteinase exists predominantly as a monomer. 12 The K D for dissociation of the SARS-CoV dimer has been reported as about 100 mM 18 or 227 mM, 17 although a much lower value has also been determined. 42 Dynamic behavior of the catalytic site Residues Cys145 (Cys144 in TGEV and HCoV 229E M pro ) and His41 constitute the catalytic dyad of coronavirus main proteinases. 8,12,13 There is a long-standing debate over the ionization state of the Cys/His active-site residue pair in cysteine proteinases. Polgar 19 has shown that it exists as a thiolate/imidazolium pair in papain-like proteinases, where the thiolate ion can be stabilized by the positive end of the permanent dipole of a long (24 residue) a-helix. However, coronavirus main proteinases (domains I/II) and picornavirus proteinases have a fold that is completely different from papain; instead, it resembles that of the serine proteinase, chymotrypsin, but with an active-site serine-to-cysteine substitution. As a consequence, there is no long a-helix available that could stabilize the thiolate anion, and indeed, Polgar 20 has proposed that rhinovirus 2A proteinase contains a hydrogen-bonded thiol/imidazole pair.\\n\\nHuang et al. 21 have replaced the catalytic Cys residue by Ser in SARS-CoV M pro and were able to detect residual proteolytic activity (at pH 7.0) for the mutated enzyme. This observation appears to support the idea that the nucleophile in this proteinase is the uncharged thiol, with His41 acting as a general base. Therefore, Cys145 and His41 of the catalytic dyad were left uncharged in all MD simulation runs in this work. In the various crystal structures of SARS-CoV M pro , the cysteinehistidine distance is between 3.6 Å and 3.9 Å . The cysteine-histidine distance of SARS-CoV M pro as a function of simulation time was monitored during the MD calculation and found to fluctuate in the range of 3.5-4.5 Å . In a recent MD simulation of a homology model for the SARS-CoV M pro monomer built on the basis of the TGEV M pro crystal structure, Pang 22 obtained distances of 1.8-3.3 Å between the S g atom of Cys145 and the N 32 of His41. The lower value is definitely by far too short and would actually correspond to a covalent S-N bond between the two residues. The values obtained in our simulations certainly agree much better with the experimental observations made by X-ray crystallography.\\n\\nIt was shown that coronavirus main proteinases do not possess a third catalytic residue at the canonical position, 8,12,13 at variance with most other proteinases, including picornavirus 2A and 3C proteinases, chymotrypsin, and papain, which have Asp, Glu or Asn as the third component of the catalytic triad. In the crystal structures of the coronavirus M pro s, a buried water molecule is found in the position that would normally be occupied by the side-chain of the third member of the catalytic triad. 8,12,13 The water molecule forms three hydrogen bonds with His41, His163/164 (or Gln163, in HCoV 229E M pro ), and Asp186/187 ( Figure 4; where two residue numbers are given, the first refers to TGEV and HCoV 229E, and the second to SARS-CoV). This hydrogen-bonding system remains intact almost over the entire simulation time, with little fluctuation. At no time during our simulations did residue Asp187 change its conformation to replace the water molecule and form a direct interaction with His41; instead, during most of the simulation time, the aspartate residue remained engaged in a salt-bridge with Arg40 (see Figure 4), thereby bridging domains I and II of the proteinase. However, in his recent MD simulation on a homology model of the SARS-CoV M pro monomer in complex with a peptide substrate, Pang 22 observed that Asp187 did change its conformation and interacted with His41, thereby replacing the water molecule and forming a catalytic triad. He proposed that the presence of a substrate with a P1 0 residue would shield the catalytic triad from solvent so that the conformational change could take place. Since we do not have a substrate in our simulations of the M pro dimer, we cannot comment on this observation; unfortunately, Pang\\'s report neither mentions the charge of His41 in the simulation nor the fate of Arg40, which would lose its interaction partner as a consequence of the reorientation of Asp187.\\n\\n\\npH-dependent conformational changes\\n\\nA major goal of the present study was the investigation of the pH-dependent conformational changes in the substrate-binding site of the SARS-CoV M pro that had been observed in the crystal structures. 13 To this end, we determined the pH-activity curve of our SARS-CoV M pro preparation, using an HPLC-based peptide cleavage assay that made use of differences in the fluorescence of substrate and products. The substrate was the pentadecapeptide SWTSAVLQSGFRKWA. This peptide corresponds to the N-terminal autocleavage site of the SARS-CoV M pro , with the exception of the P7 Ile, which had been replaced by Trp, and the P6 0 Met, which had also been replaced by Trp. The fluorescence emission of the two tryptophan residues at 353 nm was used to quantify substrate and products. The k cat /K M of our enzyme preparation with this substrate was 61.9(G1.5) mM K1 min K1 and thus about sixfold higher than what was reported by Fan et al., 18 albeit for a shorter peptide substrate. We found the pH-activity curve to be bell-shaped, with the maximum enzymatic activity at pH 7.0 (see Figure 5). This finding agrees reasonably well with that by Huang et al., 21 who reported an activity maximum at pH 7.4 using a colorimetric cleavage assay.  \\n\\n\\nSARS-CoV Main Proteinase\\n\\nThe four different MD simulations of the SARS-CoV M pro dimer described here correspond to the various pH-dependent protonation states of residues His163 and His172 in the substratebinding site (see Table 2). His163, an absolutely conserved residue at the bottom of the S1 specificity pocket, has been shown to form a hydrogen bond between its N 32 atom and the side-chain carbonyl oxygen of the P1 glutamine residue of the substrate (Figure 6(a), right image). 8,13 It is important for maintaining the absolute specificity of the M pro for glutamine in the P1 position of the substrate that His163 is not protonated; a positive charge of this histidine in an otherwise largely hydrophobic environment would probably enable glutamate (in addition to glutamine) to enter the S1 pocket. Near neutral pH and above, the protonation of His163 is prevented by a hydrophobic interaction with the aromatic ring of Phe140, and also by a strong hydrogen bond between the hydroxyl group of Tyr161 (hydrogen donor) and His163 N d1 . Yang et al. 13 had proposed that the conformational changes of the SARS-CoV M pro observed in one monomer (B) but not the other (monomer A) in the dimer crystallized at pH 6.0 could be caused by protonation of His163 at this pH, which is presumably close to the pK of this residue. The main elements of the observed conformational transition (see Figure 6(a), left and middle images) from monomer A (His163 unprotonated) to monomer B (His163 protonated) and their interpretation by Yang et al. 13 were as follows.\\n\\n(1) The side-chain of Phe140 retracts itself from the interaction with His163, presumably due to the protonation of the latter. (2) Since Phe140 is part of the oxyanion loop, this segment (residues 138-145) changes its structure significantly. In the resulting conformation, the loop is no longer able to stabilize the tetrahedral transition state of the proteolytic reaction. (3) In order to neutralize the positive charge on His163, Glu166, which along with His172 forms a wall of the S1 pocket in the conformation observed at pH 7.6 as well as in molecule A of the pH 6.0 dimer, moves \"inward\" towards His163, thereby breaking its interactions with His172 and the amino terminus (residue Ser1) of the other monomer (A) in the dimer. (4) When the amino terminus of molecule A loses its interaction with Glu166 of molecule B, the hydrogen bonds between Ser1 and the main chain of Phe140 are also lost. This contributes to the conformational change of the oxyanion loop, residues 138-145.\\n\\nAs a consequence of these concerted movements, the proteinase is presumably inactivated, owing to the oxyanion loop no longer having a competent conformation, and to Glu166 blocking the S1 substrate-binding pocket. This alternative, inactive conformation appears to be quite stable, as it even persisted upon addition of a substrate-analogous inhibitor that was observed to covalently bind to the enzyme in its inactive conformation but was not able to induce the active form and penetrate the S1 pocket. 13 The fact that the inactivating conformational changes occur only in one of the two molecules in the monoclinic crystal structure of the SARS-CoV M pro dimer is in agreement with the proteolytic activity of the enzyme being about 50% at pH 6.0 (see Figure 5). This pH value is presumably close to the pK value of His163, a notion that is confirmed by the fact that in our new tetragonal crystal form of the SARS-CoV M pro , which was obtained at a pH value slightly below pH 6.0, both monomers adopt the inactive conformation, with the amino terminus of the other protomer in the dimer not interacting with Phe140 and Glu166. This is also the case for the other new crystal form, which belongs to the orthorhombic system and was obtained at pH 6.6, i.e. again near the presumable pK of His163. However, it should be noted that both these new crystal forms contain a SARS-CoV M pro dimer that is created by crystallographic symmetry from identical monomers; thus, there is no option for the dimer to adopt different conformations in its monomers. Interestingly, the pH-dependent conformational switch can apparently also occur in the orthorhombic form, since an isomorphous crystal structure recently deposited at the Protein Data Bank (PDB) (code 2BW6) but not yet published has both monomers in the active conformation, even though it was obtained at pH 6.5 (J. Lescar, personal communication).\\n\\nThe other histidine residue involved in the S1 specificity pocket, His172, is also subject to protonation/deprotonation depending on pH. As it is engaged in a salt-bridge with Glu166 at the outer wall of the site (Figure 6(a)), its pK value should be elevated by about two units, compared to isolated histidine residues. 23 Yang et al. 13 had also determined the X-ray structure of the SARS-CoV M pro after equilibration of the crystals at pH 8.0 and described it as being very similar to the structure at pH 7.6, i.e. both monomers in the dimer were in the active conformation. However, when we closely inspected their pH 8.0 structure, we noticed that there was no salt-bridge any longer between His172 and Glu166 in either of the two monomers, while the interaction between Glu166 of monomer A and the amino terminus of the other monomer (B) in the dimer still existed (see Figure 6(b) and (c)). As a consequence, Glu166 should have more freedom to move at pH 8.0 than at pH 7.6, and one motivation for our MD simulations was to find out whether it makes use of this potential.\\n\\nAny interpretation of crystal structures making use of protonation states has to be seen in light of the inability of X-ray crystallography (except at resolutions better than 1 Å ) to directly determine the positions of hydrogen atoms because of their low scattering power for X-rays. Therefore, one purpose of the MD study presented here is to Figure 6. The S1 binding pocket and the oxyanion loop (residues 138-145, first two residues not shown) as revealed by X-ray crystallography (Yang et al. 13 and this work, monoclinic crystal form). Left panel: monomer A; right panel: monomer B. Residues of the parent monomer are shown in light blue, with the exception of Glu166, which is red. The Nterminal residues of the other monomer in the dimer are shown in dark blue. (a), (b) and (c) The crystal structures at pH 6.0, pH 7.6 and pH 8.0, respectively. 13 To visualize substrate binding to the enzyme, an additional panel is included in (a) (far right) showing the P1 Gln residue (green) of a substrate-analogous inhibitor as bound in the S1 specificity pocket. 13 Note the interaction with His163 at the distal end of the P1 glutamine. examine whether protonation/deprotonation of His163 and His172 would really lead to the conformational rearrangements seen in the crystal structures.\\n\\nOur MD simulations at the different pH values (Figure 7) started with the crystallographic coordinates of the structure determined at that particular pH, with the protonation state of His163 and His172 as indicated in Table 2. Several key distances indicative of conformational changes were monitored throughout the simulation: (1) Glu166-His163; (2) Glu166-His172; (3) Glu166-Ser1 amide nitrogen of the other monomer; and (4) Phe140 (center of mass of phenyl ring)-His163 (center of mass of imidazole ring) (Figure 8). At the outset of the pH 6.0 MD simulation (Figure 7(a)), Glu166 does not orient towards the non-protonated His163 in protomer A, but is stably oriented towards the charged His163 in protomer B. The interaction between Glu166 and His172 is a stable ion pair in the A chain during the first three nanoseconds. Thereafter, the distance between these two residues fluctuates up to about 8 Å and then back to form a hydrogen bond at about 8.5 ns. We analyzed the intermediate structure and found that sometimes Glu166 can form hydrogen bonds with the amides of Gly143, Ser144 or Cys145, i.e. the hydrogenbonding donors of the oxyanion hole. These hydrogen bonds are not stable throughout the simulation, but form and break occasionally. In contrast, in protomer B, Glu166 adopts a relatively stable intermediate position where it interacts simultaneously with His163 and His172 (both of which are protonated). His163 of protomer A makes a weak hydrophobic interaction with Phe140 for most of the simulation time. The amino terminus of the other protomer (B) in the dimer forms a saltbridge with Glu166 of protomer A and hydrogen bonds with the main chain of Phe140, although this is clearly getting weaker after about 3 ns into the simulation. In molecule B, in contrast, Glu166 does not interact with Ser1A and is instead oriented towards the now protonated His163, thereby blocking the S1 pocket. Furthermore, Phe140B stays away from His163B (as in the X-ray structure), and the oxyanion hole collapses. Thus, the MD simulation at pH 6.0 nicely confirms what has been seen in the crystal structures 13 and supports the interpretation made on this basis (compare Figures 6(a) and 7(a)).\\n\\nSimilar analyses were carried out for the pH 7.6 and pH 8.0 MD simulations (Figures 7(b) and (c), and 8(b) and (c)). At pH 7.6, Glu166 is more stably oriented towards (protonated) His172 in both monomers, while the distance between the phenyl ring of Phe140 and the imidazole ring of (nonprotonated) His163 in protomer B is relatively large compared to the X-ray structure (Figures 7(b) and  8(b)). It is found that this is due to the fact that in the initial structure, these two rings are not stacked, so their interaction may be weaker than the stack in monomer A at pH 6.0. At pH 8.0, Glu166 is more flexible with no hydrogen bonds with either His163 Figure 7. The S1-binding pocket and the oxyanion loop (residues 138-145, first two residues not shown) in the MD simulations (snapshots after 10 ns). (a), (b), (c) and (d) pH 6.0, pH 7.6, pH 8.0 and pH 5.0, respectively. Note that interaction of Glu166 (red) with His172 (to the left) constitutes the active conformation of the SARS-CoV M pro , whereas its interaction with His163 (to the right) blocks the S1 pocket (compare also Figure 6(a), middle panel) and therefore leads to inactivation of the enzyme. or His172 (Figures 7(c) and 8(c)). This is likely due to the fact that His172 as well as His163 are nonprotonated at this pH. For a significant part of the simulation time, Glu166 blocks the entry of substrate to the S1 pocket, thus explaining why the enzymatic activity is decreased at this pH (see Figure 5).\\n\\nAt the outset of the pH 5.0 MD simulation (Figures 7(d) and 8(d)), protomer A is in the active conformation as in the starting model, the pH 6.0 X-ray structure, so that at first, Glu166 is oriented towards His172, and relatively far from His163.\\n\\nAfter about 1 ns, Glu166 reorients to the protonated His163. During most of the simulation time, it is stable in this orientation. As for protomer B, the distance between Glu166 and His163 fluctuates during the first 6 ns, and then tends to be a stable interaction. From the distance between Glu166 and Ser1 of the other chain, it can be seen that this has a very similar tendency in the two chains. Thus, after 10 ns, the two chains have a similar conformation, which is the inactive conformation (Figure 7(d)).\\n\\nOur molecular dynamics simulations have confirmed that the protonation states of His163  a First number, X-ray structure; second number, MD simulation after 10 ns. pH 5.0, distances for the crystal structure are from the tetragonal form grown at pH 5.9. Other pH values, distances for the crystal structures are derived from the monoclinic structures determined by Yang et al. 13 Distances corresponding to hydrogen-bonding or ion-pair interactions are printed in bold. Note that interactions of Glu166 with His172 lead to activation of the enzyme, whereas those with His163 lead to inactivation. Other key distances are displayed in Figure 8. and His172 are important, since they can affect the conformation of the binding pocket, and thereby control the enzymatic activity. In conclusion, then, the two flanks of the bell-shaped pH-activity curve ( Figure 5) are governed by the protonation of His163 (low-pH side) and the deprotonation of His172 (high-pH side).\\n\\nIntriguingly, a comparison of the key distances after 10 ns of simulation time to the corresponding X-ray structures shows very good qualitative agreement (Table 3; also compare Figures 6 and 7). Most importantly, where there is a hydrogenbonding or ion-pair interaction in the crystal, it is almost invariably also found in the 10 ns snapshot, and where there is not, it is absent from the simulation as well. This is important, because formation and disruption of the hydrogen bonds/ ion pairs involving His163 and His172 depend on the protonation state of these side-chains. Thus, the very similar interaction pattern seen by X-ray crystallography (which cannot directly determine hydrogen positions) and MD simulations (which make full account of hydrogen atoms bound to polar atoms) lends strong support to the conclusions drawn from the crystal structures of SARS-CoV M pro by Yang et al. 13 In order to further investigate the influence of the conformational differences between the active and the inactive conformation on the shape of the substrate-binding side, we monitored the volume of the binding pocket every 50 ps during the simulations. Figure 9 shows the results of this analysis for the simulation at pH 6.0. During the first 2 ns, the volume of the pocket in protomer A shrunk significantly. Afterwards, the system appeared to have reached equilibrium and the volume of the pocket fluctuated around 800 Å 3 . In contrast, the corresponding volume in protomer B fluctuated around 600 Å 3 from the beginning. The significantly smaller value indicates that substrate binding is not favored in this conformation.\\n\\n\\nWhy is the SARS-CoV M pro monomer inactive?\\n\\nFan et al. 18 have proposed that the dimer is the enzymatically active species of the SARS-CoV M pro , since the specific activity increased linearly with enzyme concentration. Dimerization of the proteinase occurs mainly through the interactions between the helical domain III of each monomer, and through hydrogen bonding between the aminoterminal residues of one monomer and residues near the S1 subsite of the other monomer, in particular Phe140 and Glu166. 8,12,13 The N-terminal residue (Ser1 in SARS-CoV) has been proposed to keep both the oxyanion hole and Glu166 of the other monomer in a catalytically competent conformation; Yang et al. 13 coined the term of an \"N-finger\" that switches on the activity of the other monomer. In agreement with this, Anand et al. 12 observed in the TGEV M pro system that deletion of the first five residues from the N terminus abolished peptidolytic activity of the enzyme almost entirely. This is not due to a destruction of the dimer, since the SARS-CoV M pro still forms a dimer when the first seven residues from the amino terminus are missing. 17 Thus, the amino terminus, as such, is essential for enzymatic activity of the dimer. Accordingly, we carried out preliminary 10 ns MD simulations of the monomer, starting from the active conformation as found in monomer A of the crystal structure determined at pH 6.0. Intriguingly, these resulted in a collapse of the oxyanion hole because of the missing hydrogen bond between the N terminus of the other chain and the main chain of Phe140. Also, not held in position by the same N terminus, Glu166 tended to move towards His163 and block the entry of the S1 pocket, at least during part of the simulation time. These observations clarify the role of the N terminus of the other monomer in the dimer: it is needed to maintain the proper shape of the substrate-binding site, including the oxyanion hole, of the \"parent\" monomer.\\n\\n\\nConclusions\\n\\nThe SARS coronavirus main proteinase (M pro ) shows pH-dependent conformational flexibility, in particular around the substrate-binding site. We have assessed this flexibility by multiple X-ray structures (in the original crystal form as well as in two new crystal forms) and by MD simulations. The latter take into account different protonation states of two histidine residues (His163 and His172) in the substrate-binding site. Overall, we find good agreement between the flexibility data derived from the multiple crystal structures, from atomic temperature factors, and from the MD calculations. The catalytic dyad of Cys145 and His41 is confirmed to exist throughout the MD simulation; at no time is it complemented by a third residue such as Asp187. The conformation and the volume of the S1 binding pocket are controlled by the protonation states of the two histidine residues that are part of the pocket and we propose that the flanks of the bell-shaped pH-activity curve are governed by the protonation of His163 (on the low-pH side) and the deprotonation of His172 (on the high-pH side). Intriguingly, in our MD simulation of the SARS-CoV M pro monomer, the S1 pocket spontaneously collapses and adopts the inactive conformation that is also seen upon protonation of His163 in the dimeric structure. We conclude that the amino terminus of the other monomer in the dimer plays an important role for the enzyme\\'s activity by maintaining the shape of the oxyanion loop and of the S1 pocket through hydrogen bonds.\\n\\n\\nMaterials and Methods\\n\\nCrystallization and X-ray structure determination SARS-CoV M pro was produced in large quantities as described. 8,13 Crystals belonging to monoclinic space group P2 1 were grown at pH 6.0 using the published conditions. 13 Under the same conditions, especially at slightly lower pH (5.9) and higher (up to w20%) concentrations of polyethylene glycol 6000, a new tetragonal crystal form was observed, often in the same droplets where the monoclinic crystals appeared. An additional screen for improving the crystallization conditions was carried out using the \"Anion Suite\" of Nextal Biotechnologies (Montreal, Canada) and condition no. 36 (1.2 M sodium malonate in 0.1 M Mes, pH 6.5) yielded orthorhombic crystals belonging to space group P2 1 2 1 2. These were optimized at 0.7 M sodium malonate (pH 6.6). This crystal form is similar, if not identical, to the one recently described by Hsu et al. 14 Since coordinates of the structure described by these authors are not yet available, we solved the structure of the orthorhombic crystal form as well. To some crystallization setups, inhibitors were added that had been identified in an in silico screening effort 9 and subsequently synthesized or purchased. In other cases, crystals of the free enzyme were soaked in solutions containing the inhibitor in DMSO. For several of these cases, X-ray structure analysis showed that the inhibitors had not bound to the proteinase. Therefore, the resulting multiple structures of the free enzyme were used to analyze the flexibility of the M pro . Diffraction data were collected at 100 K at the Joint University of Hamburg/IMB Jena/EMBL synchrotron beamline X13 at DESY, Hamburg, Germany, at a wavelength of w0.80 Å . Intensities were measured by a MarCCD detector and processed using the HKL suite. 24 Crystal structures were determined by molecular replacement and difference Fourier methods, using the original X-ray structure for the SARS-CoV M pro crystallized at pH 6.0 (PDB code 1UJ1 13 ) as the initial model. CNS 25 and, in later stages of refinement, REFMAC 26 were employed for X-ray structure refinement. Electron density interpretation and model building were performed using the computer graphics programs O 27 and XtalView. 28 \\n\\n\\nMolecular dynamics simulations of the M pro dimer\\n\\nThe starting structures at the different pH values were taken from the Protein Data Bank (PDB entry code 1UJ1 at pH 6.0, 1UK3 at pH 7.6, and 1UK2 at pH 8.0). 13 In addition to simulations of the structural dynamics of the M pro at these pH values, calculations were also carried out at more acidic pH (pH 5.0). Since a crystal structure at a pH below 6.0 was not available at the time when these simulations were carried out, we used 1UJ1.pdb (pH 6.0) as the initial structural model, but with all histidine residues in the substrate-binding site (except His41) fully protonated. Prior to the simulations at pH 5.0, pH 6.0, and pH 7.6, residues SerA1 and GlyA2, which had not been visible in the electron density maps, 13 were modeled using the molecular modeling software Sybyl 6.8 (Tripos Associates, St Louis, MO, 2000). The modeled conformation at the beginning of each simulation was similar to that of residues SerB1 and GlyB2 of the other polypeptide chain in the dimer, which did exhibit welldefined electron density. The program package Macro-Dox version 3.2.2 29 was used to assign the titratable residues in the protein. The resulting charges of ionizable groups were appropriate for all four pH values studied, with all Arg and Lys residues protonated, and all Asp and Glu deprotonated. The protonation state of histidine residues was adopted from the MacroDox results, with the exception of His41 (see below) and of His163 and His172, which are associated with the S1 substratebinding subsite and are at the focus of the present study. Both the His163 residues of the A and the B chain of the M pro dimer were protonated in the simulation at pH 5.0, and unprotonated at pH 7.6 and 8.0. At pH 6.0, His163 was deprotonated in the A chain of the dimer, and protonated in the B chain. His172 of both chains was protonated at pH 5.0, pH 6.0 and pH 7.6, but deprotonated in both chains at pH 8.0 (Table 2). Cys145 and His41 of the catalytic dyad were uncharged in all simulations, in agreement with the recent findings by Huang et al. 21 All simulations were performed using the GROMACS (version 3.1.4) package. 30,31 The GROMOS87 force field 32 was used with modifications as suggested by van Buuren et al. 33 and explicit hydrogen atoms in aromatic rings. 34 Electrostatic interactions between charged groups at a distance less than 9 Å were calculated explicitly, longrange electrostatic interactions were calculated using the particle-mesh Ewald method 35 with a grid width of 1.2 Å and a fourth-order spline interpolation. A cutoff distance of 9 Å was applied for Lennard-Jones interactions. To maintain the system at a constant temperature of 300 K, a Berendsen thermostat 36 was applied using a coupling time of 0.1 ps. The pressure was held at 1 bar, with a coupling time of 1.0 ps. The isothermal compressibility was 4.5!10 K5 bar K1 for water simulations. The time step was set as 2 fs. All bond lengths including hydrogen atoms were constrained by the LINCS algorithm. 37 The simulation cell was a rectangular periodic box with a minimum distance of 10 Å between the protein and the box walls, so that the protein would not interact directly with its own periodic image, given the cutoff. The protein was hydrated in a box containing simple point charge (SPC) water molecules. 38 A 100 mM NaCl solution was used, alongside a number of counterions to neutralize the total charge of the system. At the end, each system contained about 75,000 atoms.\\n\\nAll systems were minimized using the steepest descent method until the convergence value of 10 kJ mol K1 Å K1 was reached. When restraints were required on, e.g. protein atoms, a harmonic potential with a force constant of 0.1 kJ mol K1 Å K2 was applied. The simulation was initiated by dynamic equilibration of solvent molecules at 300 K for 100 ps, with velocity reassignment from a Maxwellian distribution of 0.2 ps intervals and the protein structure fixed. This was followed by fixing the protein main chain, and then the C a atoms, in restraint dynamics equilibration runs of 20 ps each. Afterwards, dynamic equilibration of the entire system (solute and solvent) was performed at 300 K for 20 ps. Following the equilibrations, four 10 ns MD simulation runs were performed, with all trajectories sampled every 1.0 ps.\\n\\n\\nMolecular dynamics simulations of the M pro monomer\\n\\nSince protomer A of the M pro dimer as revealed by X-ray crystallography (pH 6.0; PDB code 1UJ1) was assumed to be in the active conformation, 13 it was selected as the reference structure in the MD simulation of the SARS-CoV M pro monomer. The MD simulations were carried out as described above for the M pro dimer. The simulation cell was a rectangular periodic box with a minimum distance of more than 7.5 Å between the protein and the box walls. To neutralize the system, 65 water molecules were replaced by 31 Cl and 34 Na ions. In total, the simulation system contained 2989 solute atoms and 11,665 solvent molecules embedded in a 59 Å !99 Å !68 Å box. Initially, all water molecules and ions, with the whole protein fixed, were energyminimized by steepest descent until the convergence value of 10 kJ mol K1 Å K1 was reached. Afterwards, minimizations were performed continuously on the protein by first fixing the main chain and subsequently the C a atoms. After the restraint dynamics equilibration, a 10 ns MD simulation was performed.\\n\\n\\nCalculation of binding site volume\\n\\nThe volume of the substrate-binding pocket was calculated using SURFNET. 39 This program first selects all atoms within 7 Å around a ligand (in this case, the peptidyl chloromethylketone inhibitor used by Yang et al. 13 ) and then introduces spheres with a diameter ranging from 4 Å to 10 Å into the space between any two non-hydrogen atoms of the protein. The volume occupied by such spheres is then estimated.\\n\\n\\nDetermination of enzymatic activity\\n\\nThe pentadecapeptide SWTSAVLQSGFRKWA was used as a substrate in an HPLC-based cleavage assay. This peptide corresponds to the N-terminal autocleavage site of the SARS-CoV M pro , with the exception of the P7 Ile, which had been replaced by Trp, and the P6 0 Met, which had also been replaced by Trp. The concentrations of SARS-CoV M pro and substrate were 0.83 mM and 0.55 mM, respectively. The reactions were carried out at different pH values in 36 mM buffer solution. The buffer was sodium acetate for pH values between 4.5 and 5.6, bis-Tris for pH 5.8-7.0, bis-Tris-propane for pH 7.0-9.4. The enzymatic activity was found to be reduced by 20% in bis-Tris-propane buffer and this was corrected for. Reactions were run for 15 min and then stopped by addition of 0.1% trifluoric acid. After 5 min cooling on ice, samples were centrifuged and 90 ml of the supernatant were mixed with 60 ml of HPLC sample buffer. Separation of products and substrate was carried out using a reversephase (RP) HPLC column (Jupiter 4m Proteo 90A; Phenomenex) and a linear gradient (1%-90%) of acetonitrile in 0.1% trifluoroacetic acid. Detection of products and substrate was achieved using a fluorescence detector (RF1002, Gynkotec) at 353 nm.\\n\\nFor determination of the enzyme kinetics, substrate concentration was varied between 0.1 mM and 1.75 mM, at a constant enzyme concentration of 0.75 mM. Substrate and enzyme were incubated in 25 mM bis-Tris-HCl buffer (pH 7.0) at 25 8C. Reaction aliquots were removed at different times up to 12 min, and analyzed by RP-HPLC as described above.\\n\\n\\nProtein Data Bank accession codes\\n\\nThe coordinates and structure factors, respectively, are available from the RCSB Protein Data Bank under PDB codes 2BX3 and r2bx3sf (tetragonal form), and 2BX4 and r2bx4sf (orthorhombic form).\\n\\nFigure 1 .\\n1Structure of the (a) monomer and (b) dimer of SARS-CoV M pro . (a) Domains I (light blue) and II (green) each contain a six-stranded b-barrel and domain III (orange) is composed mainly of a-helices. The amino and the carboxy terminus are marked by a blue and an orange sphere, respectively. The flexible loops L1, L2, and L3 (red) comprise residues 138-145 (the oxyanion-binding loop), 165-172, and 185-200, respectively. (b) a-Helices are red and b-strands are light blue. The amino and the carboxy termini are marked by blue and orange spheres, respectively. Dimerization is mainly due to interactions between the helical domains III of each monomer (top). (c) Superimposition (in stereo) of the C a backbone as determined in three different crystal forms. Blue, monoclinic form; red, tetragonal form; green, orthorhombic form. (a) and (b) were prepared by MOLSCRIPT,\\n\\nFigure 2 .\\n2Time dependence of the RMSDs from the starting structure of the SARS-CoV M pro dimer for C a atoms during the 10 ns MD simulation. (a), (b), (c), and (d) correspond to pH 6.0, pH 7.6, pH 8.0, and pH 5.0, respectively, shown as 10 ps averages.\\n\\nFigure 3 .\\n3Residue fluctuations for the SARS-CoV M pro . (a) and (b) Atomic fluctuations of (a) chain A and (b) chain B over the 10 ns equilibrium simulation at pH 6.0. (c) and (d) Mean atomic deviations (hri values) computed from the experimentally derived B factors using the equation hDr 2 i iZ 3B i =ð8p 2 Þ, for (c) chain A and (d) chain B in the SARS-CoV M pro monoclinic crystal structure (1UJ1.pdb); the corresponding values derived from the new crystal structures are shown in (d) for the tetragonal form (red) and the orthorhombic form (green). (e) and (f) RMSD for (e) monomer A and (f) monomer B for five independent monoclinic crystal structures of SARS-CoV M pro at pH 6.0, fitted to 1UJ1.pdb, and colored black, yellow, cyan, blue and magenta, respectively. The values for the new crystal structures are shown in (f) for the tetragonal form (red) and the orthorhombic form (green).\\n\\n\\n(f). These comparative data show a reasonably good agreement between the residue fluctuations derived from the MD simulations (Figure 3(a) and (b)) and the two sets of values obtained from the X-ray determination (Figure 3(c)-(f)). The RMS profiles reveal the most flexible parts of the proteinase, which are the loops L1 (residues 138-145), L2 (residues 165-172) and L3 (residues 185-200) around the substrate-binding pocket (as shown in\\n\\nFigure 4 .\\n4Conformations of the catalytic site at pH 6.0. (a) and (b) Monoclinic X-ray structure, protomers A and B, respectively; (c) and (d) 10 ns snapshot of the MD simulation; protomers A and B, respectively. Water molecules are indicated by W. Some key distances (not necessarily hydrogen bonds) are indicated by broken lines and values given in Å . Note the salt-bridge between Asp187 and Arg40 (see the text).\\n\\nFigure 5 .\\n5pH-activity curve for SARS-CoV M pro with a pentadecapeptide substrate (see the text for details).\\n\\nFigure 8 .\\n8Some characteristic distances in the MD simulations of the SARS-CoV M pro dimer at (a) pH 6.0, (b) pH 7.6, (c) pH 8.0, and (d) pH 5.0. For each simulation, the distance between Glu166 and His163, Glu166 and His172, Glu166 and Ser1(N) of the other monomer in the dimer, and Phe140 (center of mass of phenyl ring) and His163 (center of mass of imidazole ring) are shown. The shorter of the two distances to the carboxylate oxygen atoms of Glu166, O 31 and O 32 , is displayed. Green, monomer A; blue, monomer B.\\n\\nFigure 9 .\\n9Volume of the binding pocket of protomers A and B at pH 6.0 during the simulation time.\\n\\nTable 1 .\\n1Crystallographic data for the three crystal forms of SARS-CoV M pro examined in this studyCrystal form \\nMonoclinic \\nTetragonal \\nOrthorhombic \\n\\nSpace groups \\nP2 1 \\nP4 3 2 1 2 \\nP2 1 2 1 2 \\npH of crystallization \\n6.0 \\n5.9 \\n6.6 \\nWavelength (Å ) \\n0.8046 \\n0.8031 \\n0.8125 \\nResolution limits (Å ) \\n2.14 \\n2.0 \\n2.8 \\na (Å ) \\n52.02 \\n69.68 \\n108.23 \\nb (Å ) \\n96.21 \\n69.68 \\n44.56 \\nc (Å ) \\n67.53 \\n100.19 \\n54.20 \\nb (8) \\n102.84 \\n90.00 \\n90.00 \\nTotal no. of reflections \\n190,084 \\n243,889 \\n47,275 \\nUnique reflections \\n33,801 \\n15,515 \\n6962 \\nCompleteness (%) \\n98.46 \\n99.7 \\n98.1 \\nRedundancy \\n3.7 \\n14.7 \\n6.9 \\nR merge (%) \\n10.7 \\n13.9 \\n7.2 \\nI/s(I) \\n10.4 \\n20.3 \\n19.6 \\n\\n\\nTable 3 .\\n3Distances between E166(O 31 /O 32 ) and H163(N 32 ), and E166(O 31 /O 32 ) and H172(N 32 )pH 5.0 \\npH 6.0 \\npH 7.6 \\npH 8.0 \\n\\nE166-H163(A) \\n2.63/2.68 a \\n6.29/5.36 \\n6.55/4.79 \\n7.14/5.33 \\nE166-H163(B) \\n2.63/2.73 \\n2.44/2.75 \\n6.23/4.47 \\n7.79/5.58 \\nE166-H172(A) \\n3.76/2.93 \\n3.25/2.75 \\n3.73/2.57 \\n5.17/4.76 \\nE166-H172(B) \\n3.76/2.75 \\n3.70/3.50 \\n3.46/2.60 \\n5.78/5.99 \\n\\n\\nSARS-CoV Main Proteinase\\nAcknowledgementsWe are grateful to Walter Verheyen for expert technical assistance. This work was supported by the Sino-European Project on SARS Diagnostics and Antivirals (SEPSDA) of the European Commission\\nClinical progression and viral load in a community outbreak of coronavirus-associated SARS pneumonia: a prospective study. J S Peiris, C M Chu, V C Cheng, K S Chan, I F Hung, L L Poon, Lancet. 361Peiris, J. S., Chu, C. M., Cheng, V. C., Chan, K. S., Hung, I. F., Poon, L. L. et al. (2003). Clinical progression and viral load in a community outbreak of coronavirus-associated SARS pneumonia: a prospective study. Lancet, 361, 1767-1772.\\n\\nIdentification of a novel coronavirus in patients with severe acute respiratory syndrome. C Drosten, S Gunther, W Preiser, S Van Der Werf, H R Brodt, S Becker, N. Engl. J. Med. 348Drosten, C., Gunther, S., Preiser, W., van der Werf, S., Brodt, H. R., Becker, S. et al. (2003). Identification of a novel coronavirus in patients with severe acute respiratory syndrome. N. Engl. J. Med. 348, 1967-1976.\\n\\nA novel coronavirus associated with severe acute respiratory syndrome. T G Ksiazek, D Erdman, C S Goldsmith, S R Zaki, T Peret, S Emery, N. Engl. J. Med. 348Ksiazek, T. G., Erdman, D., Goldsmith, C. S., Zaki, S. R., Peret, T., Emery, S. et al. (2003). A novel coronavirus associated with severe acute respiratory syndrome. N. Engl. J. Med. 348, 1953-1966.\\n\\nMolecular mechanisms of severe acute respiratory syndrome (SARS). D A Groneberg, R Hilgenfeld, P Zabel, Respir. Res. 6Groneberg, D. A., Hilgenfeld, R. & Zabel, P. (2005). Molecular mechanisms of severe acute respiratory syndrome (SARS). Respir. Res. 6, 8-23.\\n\\nMechanisms and enzymes involved in SARS coronavirus genome expression. V Thiel, K A Ivanov, A Putics, T Hertzig, B Schelle, S Bayer, J. Gen. Virol. 84Thiel, V., Ivanov, K. A., Putics, A., Hertzig, T., Schelle, B., Bayer, S. et al. (2003). Mechanisms and enzymes involved in SARS coronavirus genome expression. J. Gen. Virol. 84, 2305-2315.\\n\\n. P S Dragovich, T J Prins, R Zhou, E L Brown, F C Maldonado, S A Fuhrman, Dragovich, P. S., Prins, T. J., Zhou, R., Brown, E. L., Maldonado, F. C., Fuhrman, S. A. et al. (2002).\\n\\nStructure-based design, synthesis, and biological evaluation of irreversible human rhinovirus 3C protease inhibitors. 6. Structure-activity studies of orally bioavailable, 2-pyridone-containing peptidomimetics. J. Med. Chem. 45Structure-based design, synthesis, and biological evaluation of irreversible human rhinovirus 3C protease inhibitors. 6. Structure-activity studies of orally bioavailable, 2-pyridone-containing peptido- mimetics. J. Med. Chem. 45, 1607-1623.\\n\\nStructure of human rhinovirus 3C protease reveals a trypsin-like polypeptide fold, RNA-binding site, and means for cleaving precursor polyprotein. D A Matthews, W W Smith, R A Ferre, B Condon, G Budahazi, W Sisson, Cell. 77Matthews, D. A., Smith, W. W., Ferre, R. A., Condon, B., Budahazi, G., Sisson, W. et al. (1994). Structure of human rhinovirus 3C protease reveals a trypsin-like polypeptide fold, RNA-binding site, and means for cleaving precursor polyprotein. Cell, 77, 761-771.\\n\\nCoronavirus main proteinase (3CL pro ) structure: basis for design of anti-SARS drugs. K Anand, J Ziebuhr, P Wadhwani, J R Mesters, R Hilgenfeld, Science. 300Anand, K., Ziebuhr, J., Wadhwani, P., Mesters, J. R. & Hilgenfeld, R. (2003). Coronavirus main proteinase (3CL pro ) structure: basis for design of anti-SARS drugs. Science, 300, 1763-1767.\\n\\nA 3D model of SARS_CoV 3CL proteinase and its inhibitors design by virtual screening. B Xiong, C.-S Gui, X.-Y Xu, C Luo, J Chen, H.-B Luo, Acta Pharmacol. Sin. 24Xiong, B., Gui, C.-S., Xu, X.-Y., Luo, C., Chen, J., Luo, H.-B. et al. (2003). A 3D model of SARS_CoV 3CL proteinase and its inhibitors design by virtual screen- ing. Acta Pharmacol. Sin. 24, 497-504.\\n\\nCoronavirus main proteinase: target for antiviral drug therapy. K Anand, H Yang, M Bartlam, Z Rao, R Hilgenfeld, Coronaviruses with Special Emphasis on First Insights Concerning SARS. Schmidt, A., Wolff, M.H. & Weber, O.Birkhäuser, BaselAnand, K., Yang, H., Bartlam, M., Rao, Z. & Hilgenfeld, R. (2005). Coronavirus main proteinase: target for antiviral drug therapy. In Coronaviruses with Special Emphasis on First Insights Concerning SARS (Schmidt, A., Wolff, M.H. & Weber, O., eds), pp. 173- 199, Birkhäuser, Basel.\\n\\nDesign of wide-spectrum inhibitors targeting coronavirus main proteinases. H Yang, W Xie, X Xue, K Yang, J Ma, W Liang, PLoS Biol. 3324Yang, H., Xie, W., Xue, X., Yang, K., Ma, J., Liang, W. et al. (2005). Design of wide-spectrum inhibitors targeting coronavirus main proteinases. PLoS Biol, 3 e324.\\n\\nStructure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an a-extra helical domain. K Anand, G J Palm, J R Mesters, S G Siddell, J Ziebuhr, R Hilgenfeld, EMBO J. 21Anand, K., Palm, G. J., Mesters, J. R., Siddell, S. G., Ziebuhr, J. & Hilgenfeld, R. (2002). Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an a-extra helical domain. EMBO J. 21, 3213-3224.\\n\\nThe crystal structures of severe acute respiratory syndrome virus main protease and its complex with an inhibitor. H Yang, M Yang, Y Ding, Y Liu, Z Lou, Z Zhou, Proc. Natl Acad. Sci. USA. 100Yang, H., Yang, M., Ding, Y., Liu, Y., Lou, Z., Zhou, Z. et al. (2003). The crystal structures of severe acute respiratory syndrome virus main protease and its complex with an inhibitor. Proc. Natl Acad. Sci. USA, 100, 13190-13195.\\n\\nMechanism of the maturation process of SARS-CoV 3CL protease. M.-F Hsu, C.-J Kuo, K.-T Chang, H.-C Chang, C.-C Chou, T.-P Ko, J. Biol. Chem. 280Hsu, M.-F., Kuo, C.-J., Chang, K.-T., Chang, H.-C., Chou, C.-C., Ko, T.-P. et al. (2005). Mechanism of the maturation process of SARS-CoV 3CL protease. J. Biol. Chem. 280, 31257-31266.\\n\\n. J Ziebuhr, G Heusipp, S G Siddell, Ziebuhr, J., Heusipp, G. & Siddell, S. G. (1997).\\n\\nBiosynthesis, purification, and characterization of the human coronavirus 229E 3C-like proteinase. J. Virol. 71Biosynthesis, purification, and characterization of the human coronavirus 229E 3C-like proteinase. J. Virol. 71, 3992-3997.\\n\\nDissection study on the severe acute respiratory syndrome 3C-like protease reveals the critical role of the extra domain in dimerization of the enzyme: defining the extra domain as a new target for design of highly specific protease inhibitors. J Shi, Z Wei, J Song, J. Biol. Chem. 279Shi, J., Wei, Z. & Song, J. (2004). Dissection study on the severe acute respiratory syndrome 3C-like pro- tease reveals the critical role of the extra domain in dimerization of the enzyme: defining the extra domain as a new target for design of highly specific protease inhibitors. J. Biol. Chem. 279, 24765-24773.\\n\\nSevere acute respiratory syndrome coronavirus 3C-like proteinase N terminus is indispensable for proteolytic activity but not for enzyme dimerization: biochemical and thermodynamic investigation in conjunction with molecular dynamics simulations. S Chen, L Chen, J Tan, J Chen, L Du, T Sun, J. Biol. Chem. 280Chen, S., Chen, L., Tan, J., Chen, J., Du, L., Sun, T. et al. (2005). Severe acute respiratory syndrome corona- virus 3C-like proteinase N terminus is indispensable for proteolytic activity but not for enzyme dimeri- zation: biochemical and thermodynamic investigation in conjunction with molecular dynamics simulations. J. Biol. Chem. 280, 164-173.\\n\\nBiosynthesis, purification, and substrate specificity of severe acute respiratory syndrome coronavirus 3C-like proteinase. K Fan, P Wei, Q Feng, S Chen, C Huang, L Ma, J. Biol. Chem. 279Fan, K., Wei, P., Feng, Q., Chen, S., Huang, C., Ma, L. et al. (2004). Biosynthesis, purification, and substrate specificity of severe acute respiratory syndrome coronavirus 3C-like proteinase. J. Biol. Chem. 279, 1637-1642.\\n\\nMercaptide-imidazolium ion-pair: the reactive nucleophile in papain catalysis. L Polgar, FEBS Letters. 47Polgar, L. (1974). Mercaptide-imidazolium ion-pair: the reactive nucleophile in papain catalysis. FEBS Letters, 47, 15-18.\\n\\nCharacterization of the active site thiol group of rhinovirus 2A proteinase. L Polgar, FEBS Letters. 481Polgar, L. (2000). Characterization of the active site thiol group of rhinovirus 2A proteinase. FEBS Letters, 481, 289-292.\\n\\n. C Huang, P Wei, K Fan, Y Liu, L Lai, Huang, C., Wei, P., Fan, K., Liu, Y. & Lai, L. (2004).\\n\\n3C-like proteinase from SARS coronavirus catalyzes substrate hydrolysis by a general-base mechanism. Biochemistry. 433C-like proteinase from SARS coronavirus catalyzes substrate hydrolysis by a general-base mechanism. Biochemistry, 43, 4568-4574.\\n\\nThree-dimensional model of a substrate-bound SARS chymotrypsin-like cysteine proteinase predicted by multiple molecular dynamics simulations: catalytic efficiency regulated by substrate binding. Y P Pang, Proteins: Struct. Funct. Bioinformat. 57Pang, Y. P. (2004). Three-dimensional model of a substrate-bound SARS chymotrypsin-like cysteine proteinase predicted by multiple molecular dynamics simulations: catalytic efficiency regulated by substrate binding. Proteins: Struct. Funct. Bioinformat. 57, 747-757.\\n\\nStabilization of ion selectivity filter by pore loop ion pairs in an inwardly rectifying potassium channel. J Yang, M Yu, Y N Jan, L Y Jan, Proc. Natl Acad. Sci. USA. 94Yang, J., Yu, M., Jan, Y. N. & Jan, L. Y. (1997). Stabilization of ion selectivity filter by pore loop ion pairs in an inwardly rectifying potassium channel. Proc. Natl Acad. Sci. USA, 94, 1568-1572.\\n\\nProcessing of X-ray diffraction data collected in oscillation mode. Z Otwinowski, W Minor, Methods Enzymol. 276Otwinowski, Z. & Minor, W. (1997). Processing of X-ray diffraction data collected in oscillation mode. Methods Enzymol. 276, 307-326.\\n\\n. A T Brunger, P D Adams, G M Clore, W L Delano, P Gros, R W Grosse-Kunstleve, Brunger, A. T., Adams, P. D., Clore, G. M., DeLano, W. L., Gros, P., Grosse-Kunstleve, R. W. et al. (1998).\\n\\nCrystallography & NMR system: a new software suite for macromolecular structure determination. Acta Crystallog. sect. D. 54Crystallography & NMR system: a new software suite for macromolecular structure determination. Acta Crystallog. sect. D, 54, 905-921.\\n\\nRefinement of macromolecular structures by the maximum-likelihood method. G N Murshudov, A A Vagin, E J Dodson, Acta Crystallog. sect. Murshudov, G. N., Vagin, A. A. & Dodson, E. J. (1997). Refinement of macromolecular structures by the maximum-likelihood method. Acta Crystallog. sect.\\n\\nImproved methods for building protein models in electron density maps and the location of errors in these models. T A Jones, J Y Zou, M Kjeldgaard, S W Cowan, Acta Crystallog. sect. A. 47Jones, T. A., Zou, J. Y., Kjeldgaard, M. & Cowan, S. W. (1991). Improved methods for building protein models in electron density maps and the location of errors in these models. Acta Crystallog. sect. A, 47, 110-119.\\n\\nXtalView/Xfit-A versatile program for manipulating atomic coordinates and electron density. D E Mcree, J. Struct. Biol. 125McRee, D. E. (1999). XtalView/Xfit-A versatile program for manipulating atomic coordinates and electron density. J. Struct. Biol. 125, 156-165.\\n\\n. S H Northrup, T Laughner, G Stevenson, Northrup, S. H., Laughner, T. & Stevenson, G. (1999).\\n\\nGROMACS: a message-passing parallel molecular dynamics implementation. H J C Berendsen, D Van Der Spoel, R Van Drunen, Comput. Phys. Commun. 91Berendsen, H. J. C., van der Spoel, D. & van Drunen, R. (1995). GROMACS: a message-passing parallel molecular dynamics implementation. Comput. Phys. Commun. 91, 43-56.\\n\\nGROMACS 3.0: a package for molecular simulation and trajectory analysis. E Lindahl, B Hess, D Van Der Spoel, J. Mol. Model. 7Lindahl, E., Hess, B. & van der Spoel, D. (2001). GROMACS 3.0: a package for molecular simulation and trajectory analysis. J. Mol. Model. 7, 306-317.\\n\\n. W F Van Gunsteren, H J C Berendsen, van Gunsteren, W. F. & Berendsen, H. J. C. (1987).\\n\\nGromos-87 Manual. Biomos BV. Groningen, The Netherlands4Gromos-87 Manual. Biomos BV, Nijenborgh 4, Groningen, The Netherlands.\\n\\nA molecular dynamics study of the decane/ water interface. A R Van Buuren, S J Marrink, H J C Berendsen, J. Phys. Chem. 97Van Buuren, A. R., Marrink, S. J. & Berendsen, H. J. C. (1993). A molecular dynamics study of the decane/ water interface. J. Phys. Chem. 97, 9206-9212.\\n\\nW F Van Gunsteren, S Billeter, A A Eising, P Hü Nenberger, P Kruger, A E Mark, Biomolecular Simulations: GROMOS96 Manual and User Guide. Biomos BV, Zü rich. Groningenvan Gunsteren, W. F., Billeter, S., Eising, A. A., Hü nenberger, P., Kruger, P. & Mark, A. E. et al. (1996). Biomolecular Simulations: GROMOS96 Manual and User Guide. Biomos BV, Zü rich, Groningen.\\n\\nParticle mesh Ewald: an Nlog(N) method for Ewald sums in large systems. T Darden, D York, L Pedersen, J. Chem. Phys. 98Darden, T., York, D. & Pedersen, L. (1993). Particle mesh Ewald: an Nlog(N) method for Ewald sums in large systems. J. Chem. Phys. 98, 10089-10092.\\n\\nMolecular dynamics with coupling to an external bath. H J C Berendsen, J P M Postma, W F Gunsteren, Dinola, J R Haak, J. Chem. Phys. 81Berendsen, H. J. C., Postma, J. P. M., Gunsteren, W. F. DiNola A & Haak, J. R. (1984). Molecular dynamics with coupling to an external bath. J. Chem. Phys. 81, 3684-3690.\\n\\nLINCS: a linear constraint solver for molecular simulations. B Hess, H Bekker, H J C Berendsen, J G E M Fraaije, J. Comput. Chem. 18Hess, B., Bekker, H., Berendsen, H. J. C. & Fraaije, J. G. E. M. (1997). LINCS: a linear constraint solver for molecular simulations. J. Comput. Chem. 18, 1463-1472.\\n\\nInteraction models for water in relation to protein hydration. H J C Berendsen, J P M Postma, W F Van Gunsteren, J Hermans, Intermolecular Forces. Pullman, B.; Dordrecht, The NetherlandsD. Reidel Publishing CompanyBerendsen, H. J. C., Postma, J. P. M., van Gunsteren, W. F. & Hermans, J. (1981). Interaction models for water in relation to protein hydration. In Intermole- cular Forces (Pullman, B., ed.), pp. 331-342, D. Reidel Publishing Company, Dordrecht, The Netherlands.\\n\\n. Sars-Cov Main Proteinase, SARS-CoV Main Proteinase\\n\\nSURFNET: A program for visualizing molecular surfaces, cavities and intermolecular interactions. R A Laskowski, J. Mol. Graph. 13Laskowski, R. A. (1995). SURFNET: A program for visualizing molecular surfaces, cavities and intermolecular interactions. J. Mol. Graph. 13, 323-330.\\n\\nMOLSCRIPT: a program to produce both detailed and schematic plots of protein structures. P J Kraulis, J. Appl. Crystallog. 24Kraulis, P. J. (1991). MOLSCRIPT: a program to produce both detailed and schematic plots of protein structures. J. Appl. Crystallog. 24, 946-950.\\n\\nThe PyMOL Molecular Graphics System. W L Delano, DeLano Scientific LLC. DeLano, W. L. (2002). The PyMOL Molecular Graphics System, DeLano Scientific LLC, San Carlos, CA, USA.\\n\\nQuaternary structure of the severe acute respiratory syndrome (SARS) coronavirus main protease. C.-Y Chou, H.-C Chang, W.-C Hsu, T.-Z Lin, C.-H Lin, G.-G Chang, Biochemistry. R. Huber43Received. received in revised form 24Chou, C.-Y., Chang, H.-C., Hsu, W.-C., Lin, T.-Z., Lin, C.-H. & Chang, G.-G. (2004). Quaternary structure of the severe acute respiratory syndrome (SARS) coronavirus main protease. Biochemistry, 43, 14958- 14970. Edited by R. Huber (Received 28 April 2005; received in revised form 24 August 2005; accepted 7 September 2005) Available online 23 September 2005\\n'"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "json.loads(fulltext_randomsample['content'][0])['text']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(len(fulltext_randomsample)):\n",
    "    # write text as file\n",
    "    with open(f'corpusid_{fulltext_randomsample.corpusid[i]}.txt', 'w') as f:\n",
    "        f.write(json.loads(fulltext_randomsample['content'][i])['text'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "annotation_sample = fulltext_randomsample[['corpusid','content']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/82/mcr3vdtj7kg2_7lr7tq4vk6m0000gn/T/ipykernel_54181/1991968635.py:1: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  annotation_sample['content'] = annotation_sample['content'].apply(lambda x: json.loads(x)['text'])\n"
     ]
    }
   ],
   "source": [
    "annotation_sample['content'] = annotation_sample['content'].apply(lambda x: json.loads(x)['text'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "annotation_sample.to_csv('annotation_sample.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "import nltk"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['corpusid_43735018.txt',\n",
       " 'corpusid_252931081.txt',\n",
       " 'corpusid_22497657.txt',\n",
       " 'corpusid_16707705.txt',\n",
       " 'corpusid_238718700.txt',\n",
       " 'corpusid_36080.txt',\n",
       " 'corpusid_6310005.txt',\n",
       " 'corpusid_248085185.txt',\n",
       " 'corpusid_258109476.txt',\n",
       " 'corpusid_262124348.txt',\n",
       " 'corpusid_231665585.txt',\n",
       " 'corpusid_244492526.txt',\n",
       " 'corpusid_220409365.txt',\n",
       " 'corpusid_256958918.txt',\n",
       " 'corpusid_184487052.txt',\n",
       " 'corpusid_213940714.txt',\n",
       " 'corpusid_18818806.txt',\n",
       " 'corpusid_11641444.txt',\n",
       " 'corpusid_2167061.txt',\n",
       " 'corpusid_226248203.txt',\n",
       " 'corpusid_253798281.txt',\n",
       " 'corpusid_239536179.txt',\n",
       " 'corpusid_17476268.txt',\n",
       " 'corpusid_219116765.txt',\n",
       " 'corpusid_251646620.txt',\n",
       " 'corpusid_253611370.txt',\n",
       " 'corpusid_119457479.txt',\n",
       " 'corpusid_248027036.txt',\n",
       " 'corpusid_17217044.txt',\n",
       " 'corpusid_252780743.txt',\n",
       " 'corpusid_77692.txt',\n",
       " 'corpusid_236968440.txt',\n",
       " 'corpusid_16006581.txt',\n",
       " 'corpusid_213186406.txt',\n",
       " 'corpusid_228066271.txt',\n",
       " 'corpusid_15989212.txt',\n",
       " 'corpusid_17146855.txt',\n",
       " 'corpusid_232101395.txt',\n",
       " 'corpusid_118416939.txt',\n",
       " 'corpusid_3193080.txt',\n",
       " 'corpusid_237556021.txt',\n",
       " 'corpusid_262995416.txt',\n",
       " 'corpusid_245576511.txt',\n",
       " 'corpusid_257198162.txt',\n",
       " 'corpusid_143423969.txt',\n",
       " 'corpusid_253244624.txt',\n",
       " 'corpusid_257312156.txt',\n",
       " 'corpusid_8391628.txt',\n",
       " 'corpusid_118636449.txt',\n",
       " 'corpusid_56102549.txt',\n",
       " 'corpusid_232479186.txt',\n",
       " 'corpusid_238239577.txt',\n",
       " 'corpusid_244711156.txt',\n",
       " 'corpusid_233563006.txt',\n",
       " 'corpusid_256911034.txt',\n",
       " 'corpusid_263285864.txt',\n",
       " 'corpusid_253399304.txt',\n",
       " 'corpusid_6331263.txt',\n",
       " 'corpusid_229316033.txt',\n",
       " 'corpusid_254736170.txt',\n",
       " 'corpusid_218689985.txt',\n",
       " 'corpusid_24941224.txt',\n",
       " 'corpusid_257058789.txt',\n",
       " 'corpusid_237503175.txt',\n",
       " 'corpusid_9229284.txt',\n",
       " 'corpusid_238749287.txt',\n",
       " 'corpusid_37123679.txt',\n",
       " 'corpusid_257856681.txt',\n",
       " 'corpusid_22501882.txt',\n",
       " 'corpusid_219456240.txt',\n",
       " 'corpusid_131773844.txt',\n",
       " 'corpusid_249677555.txt',\n",
       " 'corpusid_219528359.txt',\n",
       " 'corpusid_119489650.txt',\n",
       " 'corpusid_208335700.txt',\n",
       " 'corpusid_204748540.txt',\n",
       " 'corpusid_18143733.txt',\n",
       " 'corpusid_234022863.txt',\n",
       " 'corpusid_218908174.txt',\n",
       " 'corpusid_260580325.txt',\n",
       " 'corpusid_254895464.txt',\n",
       " 'corpusid_34977363.txt',\n",
       " 'corpusid_242921784.txt',\n",
       " 'corpusid_52282455.txt',\n",
       " 'corpusid_226327353.txt',\n",
       " 'corpusid_17292556.txt',\n",
       " 'corpusid_56330314.txt',\n",
       " 'corpusid_17763614.txt',\n",
       " 'corpusid_258171714.txt',\n",
       " 'corpusid_10574571.txt',\n",
       " 'corpusid_55367888.txt',\n",
       " 'corpusid_235823156.txt',\n",
       " 'corpusid_8695014.txt',\n",
       " 'corpusid_21057537.txt',\n",
       " 'corpusid_54758390.txt',\n",
       " 'corpusid_9174601.txt',\n",
       " 'corpusid_53210972.txt',\n",
       " 'corpusid_237504530.txt',\n",
       " 'corpusid_261339628.txt',\n",
       " 'corpusid_16270039.txt']"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "#get list of files in 'Methods' folder\n",
    "files = os.listdir('Methods')\n",
    "sentences = pd.DataFrame()\n",
    "for file in files:\n",
    "    #open file\n",
    "    with open(f'Methods/{file}', 'r') as f:\n",
    "        #get corpusid\n",
    "        corpusid = file.split('_')[1].split('.')[0]\n",
    "        #read file\n",
    "        text = f.read()\n",
    "        segments = text.split('\\n')\n",
    "        #tokenize\n",
    "        for segment in segments:\n",
    "            tokens = nltk.sent_tokenize(segment)\n",
    "            #create dataframe\n",
    "            df = pd.DataFrame(tokens, columns=['sentence'])\n",
    "            df['corpusid'] = corpusid\n",
    "            sentences = pd.concat([sentences, df])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "sentences.to_csv('sentences.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "def has_proper_noun(sentence):\n",
    "    words = nltk.word_tokenize(sentence)\n",
    "    pos_tags = nltk.pos_tag(words)\n",
    "    for word, pos in pos_tags:\n",
    "        if pos == 'NNP' or pos == 'NNPS':\n",
    "            return True\n",
    "    return False"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "sentences['proper_noun'] = sentences['sentence'].apply(has_proper_noun)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "sentences[sentences['proper_noun']].to_csv('sentences_proper_noun.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>sentence</th>\n",
       "      <th>corpusid</th>\n",
       "      <th>proper_noun</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>No methods</td>\n",
       "      <td>43735018</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Methods</td>\n",
       "      <td>252931081</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Participants and procedures</td>\n",
       "      <td>252931081</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>.</td>\n",
       "      <td>252931081</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Demographic characteristics and bereavement-re...</td>\n",
       "      <td>252931081</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Materials and Methods</td>\n",
       "      <td>16270039</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>6-8-wk-old females were used in all experiment...</td>\n",
       "      <td>16270039</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>In double labeling experiments, the PE-conjuga...</td>\n",
       "      <td>16270039</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Hybrid antibodies were transiently expressed i...</td>\n",
       "      <td>16270039</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Adoptive Transfer.</td>\n",
       "      <td>16270039</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2227 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                             sentence   corpusid  proper_noun\n",
       "0                                          No methods   43735018        False\n",
       "0                                             Methods  252931081        False\n",
       "0                         Participants and procedures  252931081        False\n",
       "0                                                   .  252931081        False\n",
       "0   Demographic characteristics and bereavement-re...  252931081        False\n",
       "..                                                ...        ...          ...\n",
       "0                               Materials and Methods   16270039        False\n",
       "1   6-8-wk-old females were used in all experiment...   16270039        False\n",
       "4   In double labeling experiments, the PE-conjuga...   16270039        False\n",
       "0   Hybrid antibodies were transiently expressed i...   16270039        False\n",
       "0                                  Adoptive Transfer.   16270039        False\n",
       "\n",
       "[2227 rows x 3 columns]"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sentences[sentences['proper_noun']==False]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "ner",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
